flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendTestUtils.java

```diff
@@ -65,9 +67,14 @@ public static String getInvalidConfigDir() {
 	}
 
 	public static void pipeSystemOutToNull() {
+		previousSysout = System.out;
```

Comment:
Doe it have to be done this way or could we make `previousSysout` final?

Reply:
That depends on whether we want the restore logic as "restore to previous stdout" or "restore to default stdout" - I guess, the latter makes most sense indeed and we do not want to encourage any other use.
-> I'll change the commits accordingly